### PR TITLE
Improve Ref Docs

### PR DESCRIPTION
**Pull request by sommersoft**
[Per Issue #566](https://github.com/adafruit/circuitpython/issues/566)

The following Learn Guide will need the 2 GitHub example code links updated:
https://learn.adafruit.com/adafruit-lsm9ds0-accelerometer-gyro-magnetometer-9-dof-breakouts?view=all#circuitpython-code

### CODE REVIEW

File: adafruit_lsm9ds0.py [hunk 1]
```diff
@@ -248,7 +265,7 @@ def read_accel_raw(self):
 
     @property
     def accelerometer(self):
-        """Get the accelerometer X, Y, Z axis values as a 3-tuple of
+        """Returns the accelerometer X, Y, Z axis values as a 3-tuple of
         m/s^2 values.
```
Comment (kattni) [inline adafruit_lsm9ds0.py]: "The accelerometer ..." Doesn't need "Returns".

File: adafruit_lsm9ds0.py [hunk 2]
```diff
@@ -269,7 +286,7 @@ def read_mag_raw(self):
 
     @property
     def magnetometer(self):
-        """Get the magnetometer X, Y, Z axis values as a 3-tuple of
+        """Returns the magnetometer X, Y, Z axis values as a 3-tuple of
         gauss values.
```
Comment (kattni) [inline adafruit_lsm9ds0.py]: Same.

File: adafruit_lsm9ds0.py [hunk 3]
```diff
@@ -289,7 +306,7 @@ def read_gyro_raw(self):
 
     @property
     def gyroscope(self):
-        """Get the gyroscope X, Y, Z axis values as a 3-tuple of
+        """Returns the gyroscope X, Y, Z axis values as a 3-tuple of
```
Comment (kattni) [inline adafruit_lsm9ds0.py]: Same.

File: adafruit_lsm9ds0.py [hunk 4]
```diff
@@ -308,7 +325,7 @@ def read_temp_raw(self):
 
     @property
     def temperature(self):
-        """Get the temperature of the sensor in degrees Celsius."""
+        """Returns the temperature of the sensor in degrees Celsius."""
```
Comment (kattni) [inline adafruit_lsm9ds0.py]: Same.

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for doing this! See inline change requests.

**Comment by sommersoft** [comment]
And here I was thinking "I'll surprise kattni and get these right"... 😄

**Comment by kattni** [comment]
You're doing a great job! There's obviously a learning curve when you run
into new things. :) You've been picking everything up amazingly! Thank you
again!

**Review by kattni — APPROVED** [review]
Excellent!

**Pull request merged**